client/src/components/Layout/LandingPage.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import {BsCircleFill} from 'react-icons/bs';
+import img1 from "../../assets/body-img1.png";
+import img2 from "../../assets/body-img2.png";
+import img3 from "../../assets/body-img3.png";
+
+const LandingPage=()=> {
+    return(
+        <div className='bg-black text-white'>
+            <div className='w-[44rem] mx-auto text-center'>
+            <h3 className='text-[2.5rem] font-bold'>Stand out with a free Business Profile</h3>
+            <p className='text-[1.25rem] pt-3 font-thin'>Turn people who find you on this website into new customers with a free Business Profile for your 
+                storefront or service area.Personalise your profile with promotion product photos, offers, posts, and more.</p>
+            </div>
+            <div  className='w-[46rem] mx-auto mt-[4rem] text-center'>
+                <h3 className='text-[2.5rem] font-bold'>First impression is the last impresion</h3>
+                <p  className='text-[1.25rem] pt-3 font-thin'>Highlight essential info and show what makes your business unique, right on your Business Profile.</p>
+            </div>
+            <div className='flex flex-row space-x-[6rem] justify-center pt-[10rem] px-[12rem]'>
+                <div className='flex-row space-x-2 inline-flex'>
```

Comment:
Add `w-[30rem] h-[30rem]`

Reply:
in which line?

Reply:
@varisha-025

Reply:
Line 19

Reply:
![image](https://user-images.githubusercontent.com/96660357/198677442-191105e0-20f1-4b2a-82b2-f938bb3b4471.png)



@varisha-025

Reply:
@varisha-025 once review your required changes pls.

Reply:
Try line 20

Reply:
Maybe because of screen difference as I have 13 inch mac.

Reply:
Shift the first card's image upwards and increase the height by 1 or 2 rem. whatever looks good

Reply:
okey

Reply:
Do this and make the PR

Reply:
all the 3 imgs should be in one online otherwise it wont look uniform to user's eye

Reply:
better i will increase the height

Reply:
Did you make it responsive?

Reply:
No I haven't. You add another issue to make it responsive.My exams have started, so I am not able to devote much time to it

Reply:
okay. force push the changes then